File: Darts/src/code/db/sanity/SanityCheckResultUnsetColumns.java
package code.db.sanity;

import java.util.ArrayList;

import code.db.AbstractEntity;

public class SanityCheckResultUnsetColumns
		extends AbstractSanityCheckResultEntities
{
	private String columnName = "";
	
	public SanityCheckResultUnsetColumns(String columnName, ArrayList<? extends AbstractEntity<?>> entities)
	{
		super(entities);
		
		this.columnName = columnName;
	}
	
	@Override
	public String getDescription()
	{
		return getEntityName() + " rows where " + columnName + " is unset";
	}

}

File: src/main/kotlin/dartzee/db/sanity/SanityCheckResultUnsetColumns.kt
package dartzee.db.sanity

import dartzee.db.AbstractEntity

class SanityCheckResultUnsetColumns(
    private val columnName: String,
    entities: List<AbstractEntity<*>>,
) : AbstractSanityCheckResultEntities(entities) {
    override fun getDescription() = "$entityName rows where $columnName is unset"
}
